docs/port-creation.md

```diff
@@ -0,0 +1,97 @@
+## Requesting a new port
+
+1. Create a new discussion with the [port request](https://github.com/catppuccin/catppuccin/discussions/new?category=port-requests) template.
+2. Fill out the discussion template with the details of the program you want to port.
+3. How the website can be themed.
+4. If you have made the port yourself, you can add a link to your fork.
+
+## Creating a new port
+
+1. Fork this repository and clone your fork.
+2. Create a new folder in the `styles` directory with the name of the program you are porting.
```

Comment:
```suggestion
2. Create a new folder in the `styles` directory with the name of the website you are porting. The name must be `lower-kebab-case` (e.g. `nixos-search` instead of NixOS Search).
```